components/homme/src/theta-l_kokkos/CMakeLists.txt

```diff
@@ -11,8 +11,8 @@ MACRO(THETAL_KOKKOS_SETUP)
   SET(TEST_SRC_DIR      ${HOMME_SOURCE_DIR}/src/test_src)
```

Comment:
IIRC this change will break homme standalone build (and won't be caught via nightlies). Someone needs to check and fix this (or leave homme's version in e3sm for now).

Reply:
@oksanaguba I ran the following script and see no issues; whatever was broken may have been fixed. Is there anything else I should do to address this concern? Thanks.
```
rm -rf CMake*
e3sm=~/SCREAMref # SCREAM master of the moment
cmake -C $e3sm/components/homme/cmake/machineFiles/chrysalis-bfb.cmake \
      -D BUILD_HOMME_THETA_KOKKOS=On \
      -D BUILD_HOMME_PREQX_KOKKOS=On \
      -D NUM_CPUS=24 \
      -D Kokkos_ENABLE_OPENMP=On \
      -D Kokkos_ENABLE_SERIAL=On \
      $e3sm/components/homme/
make -j64
ctest
```

Reply:
great, thanks, it works.